tests/dhcp_relay/test_dhcpv6_relay.py

```diff
@@ -487,3 +493,46 @@ def test_dhcp_relay_start_with_uplinks_down(ptfhost, dut_dhcp_relay_data, valida
                            "loopback_ipv6": str(dhcp_relay['loopback_ipv6']),
                            "is_dualtor": str(dhcp_relay['is_dualtor'])},
                    log_file="/tmp/dhcpv6_relay_test.DHCPTest.log", is_python3=True)
+
+
+def test_dhcp_relay_default_with_multiple_vlan(ptfhost, dut_dhcp_relay_data, validate_dut_routes_exist, testing_config,
+                                               toggle_all_simulator_ports_to_rand_selected_tor_m, # noqa F811
+                                               setup_active_active_as_active_standby,             # noqa F811
+                                               setup_multiple_vlans_and_teardown):                # noqa F811
+    '''
+        Test DHCP relay should set correct link address when relay packet to DHCP server
+    '''
+    vlans_info = setup_multiple_vlans_and_teardown
+    _, duthost = testing_config
+    skip_release(duthost, ["201811", "201911", "202106"])  # TO-DO: delete skip release on 201811 and 201911
+
+    # Please note: relay interface always means vlan interface
+    pytest_assert(len(dut_dhcp_relay_data) > 0, "No VLAN data")
+    common_dhcp_relay_data = dut_dhcp_relay_data[0]
+    for vlan_info in vlans_info:
+        vlan_name = vlan_info['vlan_name']
+        exp_link_addr = vlan_info['interface_ipv6'].split('/')[0]
+        _, ptf_port_index = random.choice(vlan_info['members_with_ptf_idx'])
```

Comment:
Can we add a log for this random result to help debug in case flaky failure happen?

Reply:
Yes, will add log.